cloudify_cli/config/parser_config.py

```diff
@@ -86,15 +88,15 @@ def validate_blueprint_argument():
         'dest': 'validate_blueprint',
         'action': 'store_true',
         'help': 'Validate the blueprint before uploading it '
-                'to the manager'
+                'to Cloudify Manager'
```

Comment:
This is the only time that you refer to the Manager as 'Cloudify Manager' and not as 'Manager'. Why?